### DISCUSSION TITLE
[Feature]: Support for Canon CR3 RAW format

### DISCUSSION THREAD

**Discussion by chandlben** [immich-app/immich#1588] [Feature Request]
### Feature detail

Would love to see support for Canon CR3 RAW files.

[Sample CR3 (from raw.pixls.us)](https://raw.pixls.us/getfile.php/4672/nice/Canon%20-%20Canon%20EOS%20RP%20-%203:2.CR3)

### Platform

Server

**Comment by clam0r** [comment]
I would also love the support of CR3 Files

**Comment by Sixth2538** [comment]
Would love to see more raw format support such as Olympus as well #1653

**Comment by uhthomas** [comment]
Resolved by #2834.
